letsdrill.rb

```diff
@@ -1,13 +1,33 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+### Using some ternaries (ternarys? ternararies?)
+
+  integer >= 90 ? "A" :
+    integer >= 80 ? "B" :
+    integer >= 70 ? "C" :
+    integer >= 60 ? "D" :
+    "F"
 
 end
 
 def shortest_string(array)
 
-  #Put your code here!
-
+  # First guess that the first array
+  # element is the shortest.
+  shortest = array[0]
+    
+  # Iterate through each string in the array
+  array.each {|string|
+  # If the string we're looking at is shorter than
+  # the shortest we've seen so far...
+      if string.length < shortest.length
+          # ...set the shortest string to this
+          # current string.
+          shortest = string
+      end
+      }
+  # Implicit return! Because we can!
+  shortest
```

Comment:
This is very easy to read. 👍 

Ruby provides many methods to built-in classes. In particular, [all these methods on `Enumerable`](https://ruby-doc.org/core-2.6.1/Enumerable.html) are available to instances of `Array`.

Consider rewriting this method to sort the input array by the length of the elements, then just plucking off the answer.

Or, if you scan through those Enumerable methods, you might find [`min_by`](https://ruby-doc.org/core-2.6.1/Enumerable.html#method-i-min_by) which basically does the whole thing for you. But what's the fun in that?

Reply:
Thanks very much for the feedback!